app/services/scholars_archive/ordered_parser_service.rb

```diff
@@ -0,0 +1,18 @@
+module ScholarsArchive
+  class OrderedParserService
+    def self.parse(labels)
+      labels ||= []
+      parsed_labels = []
+      puts labels
+      sorted_labels = labels.map{ |label| build_array(label)}.sort_by{ |array| array[1] }.map{ |array| array[0] }
+      puts sorted_labels
```

Comment:
I think we can remove the lines `puts labels` and `puts sorted_labels` too. Did you put them there to inspect the values only? @straleyb